Tests/Unit Tests/Extensions/XCTestCase+Additions.swift

```diff
@@ -25,4 +25,34 @@ extension XCTestCase {
         }
     }
     
+    /// Asserts that a `Validator` with a given input returns the expected result.
+    ///
+    /// - Parameters:
+    ///   - validator: `Validator` to test.
+    ///   - testInput: Input to have `validator` check.
+    ///   - expectedResult: The expected result for running the validator's check.
+    ///   - file: File that the assertion is being run in. Defaults to `#file`.
+    ///   - line: Line that the assertion is being run from. Defaults to `#line`.
+    ///
+    /// - Note: Currently this checks that either the actual result and expected result are either both nil or both non-nil and have the same number of elements. As `Condition` currently does not conform to equatable, further checks are not done to ensure that the conditions in the array are the same.
+    func AssertValidator(_ validator: Validator, testInput: String?, expectedResult: [Condition]?, file: String = #file, line: UInt = #line) {
```

Comment:
Just legacy, to match the `AssertCondition` naming and make it internally consistent. I'm planning to do a full stylistic sweep before 2.0 to make sure everything is following the Swift 3 API design guidelines.